homie5/formats/datetime.yml

```diff
@@ -0,0 +1,5 @@
+description: validating dateTime formats
+
+# This type does not have a specified format entry in the Homie convention
```

Comment:
Is there any value to test whether a format is provided anyway|?

Reply:
good call. I don't think the spec currently forbids to pass in a format property ofr these types. Should we add that?
wdyt @schaze ?

Reply:
oh, but this is for controllers. Not devices... makes sense. Sorry, I'm new to this :sweat_smile:

Reply:
But forward compatibility may add it in the future. So maybe we should leave it as is, and just ignore it.